src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] Linters help you to stick to best practices, instead of disabling them, it will be nice to fix the issue. Please kindly fix the issue with `eslint`.